docs/import/manage_disaster_base.js

```diff
@@ -392,11 +395,15 @@ async function verifyAsset(propertyPath, expectedColumns) {
   if (asset === '') {
     updateColorAndHover(select, 'white', '');
     return [];
-  } else if (!expectedColumns || expectedColumns.length === 0) {
-    updateColorAndHover(select, 'yellow', 'Checking...');
+  } else {
+    const checkingColumns = expectedColumns && expectedColumns.length > 0;
+    if (checkingColumns) {
+      updateColorAndHover(select, 'yellow', 'Checking columns...');
+    } else {
+      updateColorAndHover(select, 'yellow', 'Checking...');
```

Comment:
What checking do we need to do in this case? Why do we not go straight to green?

Reply:
Whether or not it exists.

This is just a refactoring of the existing code because we're getting names unconditionally now, but users were already seeing this.